source/agora/node/FullNode.d

```diff
@@ -396,7 +396,7 @@ public class FullNode : API
                 .each!(handler => handler.addresses
                     .each!((string address) {
                         auto url = Address(address); // TODO normalize
-                        this.block_header_handlers[url] = this.network.getBlockHeaderUpdatedHandler(url);   
```

Comment:
Whitespace fixes in their own commit.

Reply:
Editor did it automatically on save, I will revert